contracts/package.json

```diff
@@ -65,6 +73,10 @@
     "@rollup/plugin-commonjs": "^25.0.7",
     "@rollup/plugin-json": "^6.1.0",
     "@rollup/plugin-node-resolve": "^15.2.3",
+    "@typechain/ethers-v6": "^0.5.0",
```

Comment:
This is a big nope. Ethers V6 got rid of BigNumber. We use it everywhere on our repo. I know this is just types but let's not complicate our dependencies and end up breaking something

Reply:
yup thanks removing this